azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/NetworkInterfaceCard.java

```diff
@@ -51,6 +51,7 @@ public static NetworkInterfaceCard create(final String name,
                                              final String location,
                                              final NetworkInterfaceCardProperties properties,
                                              final Map<String, String> tags) {
-      return new AutoValue_NetworkInterfaceCard(name, id, etag, location, properties, tags == null ? null : ImmutableMap.copyOf(tags));
+      return new AutoValue_NetworkInterfaceCard(name, id, etag, location, properties,
+         tags == null ? ImmutableMap.<String, String>of() : ImmutableMap.copyOf(tags));
```

Comment:
I think we shouldn't change how the `tags` fields are configured. It is OK to have them `null`(although it is not the default practice). See: https://github.com/jclouds/jclouds-labs/pull/262#r60811082
Could you please revert the changes made to the already existing `tags`fields?